web-www/next.config.js

```diff
@@ -26,6 +26,7 @@ const nextConfig = {
       },
     ],
   },
+  transpilePackages: ['web-components'],
```

Comment:
needed to add this so the code in web-components gets transpiled (otherwise only the code in the root next.js project is)